src/main.rs

```diff
@@ -90,6 +107,8 @@ enum AppInput {
     InitialConfiguration,
     StartUpgrade,
     Restart,
+    Quit,
+    ShouWindow,
```

Comment:
Typo and order should be more logical (and matching order in the code below):
```suggestion
    ShowWindow,
    Quit,
```